src/model.ts

```diff
@@ -391,7 +392,7 @@ export default class MediaPlayerObject {
             },
             PLATFORM.SQUEEZEBOX,
           );
-        case PLATFORM.MEDIAPLAYER:
+        case PLATFORM.MEDIAPLAYER || PLATFORM.SONOS:
```

Comment:
The OR operator doesn't work as you'd think here, this case will always evaluate to the value of `PLATFORM.MEDIAPLAYER` since that is considered truthy, we should rewrite this into two cases with fall-through to the second one.

```suggestion
        case PLATFORM.MEDIAPLAYER:
        case PLATFORM.SONOS:
```